gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=0.25.0
+version=2.0.102
```

Comment:
What is this new versioning bro

Reply:
well its v2 plus some as yes

Reply:
maybe make a new versioning semantics page for polyui in contributor docs